recipes/godot_headers/all/test_package/test_package.c

```diff
@@ -0,0 +1,6 @@
+#include <gdnative/vector2.h>
+
+int main()
+  {
+  godot_vector2 vector;
+  }
```

Comment:
```suggestion
{
    godot_vector2 vector;
    return 0;
}
```

GCC <5 returns 1 by default.

Reply:
Good to know! Resolved.